authorizations.md

```diff
@@ -0,0 +1,92 @@
+# Authorizations Endpoints
+[Back to the list of all defined endpoints](endpoints.md)
+
+An authorizations is the representation of some rights that are available to a specific user (eperson) in a defined scope, eventually the whole repository (site object).
+All the authorizations are always explicitly listed regardless to how they are grant, by direct policies via groups' membership or open to everyone (anonymous users).
+
+## Main Endpoint
+**/api/authz/authorizations**
+
+As we don't have yet an use case to iterate over all the existent authorizations the main endpoint is not implemented and a 405 error code is returned according to our [general error response codes](README.md#Error codes).
+
+## Single Authorization
+**/api/authz/resourcepolicies/<:id>**
+
+Provide detailed information about a specific authorization. The JSON response document is as follow
+
+```json
+{
+  "id": [eperson_uuid-]feature_id-scope_uuid,
+  "type": "authorization"
+}
+```
+
+Attributes
+* id: the id of the authorization resource is defined by the combination of the eperson uuid (if not null), the feature id and the scope uuid joined with a minus 
+
+Exposed links:
+* eperson: link to the eperson that the authorization belong to. Can be null for authorizations grant to unlogged users
+* feature: link to the feature enabled by this authorization
+* scope: link to the object scope where this authorization apply. Not limited to DSpace objects, see [features endpoint](features.md) for more details
```

Comment:
This note about `scope` doesn't make sense to me, as it's not described further in the `features` endpoint.

Reply:
I improved the description, I mean to point to this information
https://github.com/DSpace/Rest7Contract/pull/92/files#diff-e1ec7fcfd5aa4d3ea3e990d9b707a3b4R52